code/dynamic_programming/src/longest_repeating_subsequence/longest_repeating_subsequence.cpp

```diff
@@ -0,0 +1,38 @@
+// Part of Cosmos by OpenGenus Foundation
+
+//dynamic programming || Longest repeating subsequence
+
+#ifndef LONGEST_REPEATING_SUBSEQUENCE
+#define LONGEST_REPEATING_SUBSEQUENCE
```

Comment:
Don't place header guards in a C++ file. If you want to create a library interface, put all function declarations into a separate header file with guards, and move implementations into a C++ file.